### PR TITLE
Small tweaks for python3

### PR DESCRIPTION
**Pull request by iarspider**
Most of the changes are purely cosmetic. The `github_get_file_changes.py` script is used by [GitHub-refactor-cmssw](https://cmssdt.cern.ch/jenkins/job/GitHub-refactor-cmssw/) job that was not ran since Feb 2020.

### CODE REVIEW

**Comment by iarspider** [comment]
There is one file with `python2` in hashbang - I didn't touch it since I can't find any usage of it: [checkLogFile.py](https://github.com/cms-sw/cms-bot/blob/master/checkLogFile.py)

File: github_get_file_changes.py [hunk 1]
```diff
@@ -171,9 +171,18 @@ def main():
...
+        unmodified_modules_sorted_by_time = sorted(
+            unmodified_modules_sorted_by_time, cmp=lambda x, y: cmp_f(x[1], y[1])
+        )
+    else:
+        import functools
+
+        unmodified_modules_sorted_by_time = sorted(
+            unmodified_modules_sorted_by_time,
+            key=functools.cmp_to_key(lambda x, y: cmp_f(x[1], y[1])),
+        )
```
Comment (smuzaffar) [inline github_get_file_changes.py:185]: @iarspider , if I remember correctly, this script was used when we start applying code format changes. This script was used to get the list of changed files of opened PR. There is no need to support python2 here. Just update the shibang statement to use python3 and update this code to only have python3

File: gitmergesgraph.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def load_graph(release_queue, maxNodes):
                 # connect this node with the previous one from of the same lane
                 previous_node = get_previous_node_lane(prev_node_lane, lane)
 
-            if previous_node == None:
+            if previous_node is None:
```
Comment (smuzaffar) [inline gitmergesgraph.py:75]: @iarspider , this script does not run in IB/PRs, so please update the shibang to use python3

File: scram-package-monitor-sender.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def esReportPackages(results):
     except HTTPError as e:
         print(e)
         try:
-            print(result.read())
+            print(result.read().decode())
```
Comment (smuzaffar) [inline scram-package-monitor-sender.py:36]: @iarspider , checkif ` base64.encodestring` works in python3 ... I think it is deprecated

File: update-github-hooks-ip.py [hunk 4]
```diff
@@ -16,5 +16,5 @@
 if cnt:
     system("mv %s.tmp %s" % (ip_file, ip_file))
 else:
-    system("rm -f %s.tmp %s" % ip_file)
+    system("rm -f %s.tmp" % ip_file)
```
Comment (smuzaffar) [inline update-github-hooks-ip.py:19]: for cms-bot scripts which do not need py2/py3 compatibility, please remove `from _py2with3compatibility ...` statements and directly import what is needed for python3

File: scram-package-monitor-sender.py (hunk 3 above)
Comment (iarspider) [reply scram-package-monitor-sender.py:36]: Works but deprecated since 3.1 .

**Comment by smuzaffar** [comment]
looks good, I am merging it now

**Pull request merged**